sdk/communication/Azure.Communication.CallingServer/src/Models/DtmfOptionsInternal.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Communication.CallingServer
+{
+    [CodeGenModel("DtmfOptions")]
+
+    internal partial class DtmfOptionsInternal
```

Comment:
why is this part of this PR?

Reply:
The code was already merged to master, thus it got autogenerated into the swagger and these models were generated by Autorest.